scripts/write-emails-to-disk.js

```diff
@@ -124,7 +124,10 @@ function getMailerMessageTypes(mailer) {
   var messageTypes = []
 
   for (var key in mailer) {
-    if (typeof mailer[key] === 'function' && ! /^_/.test(key) && /Email$/.test(key)) {
+    if (
+      typeof mailer[key] === 'function' &&
+      ! /^_/.test(key) && ! /^send/.test(key) && /Email$/.test(key)
```

Comment:
I am indifferent, either way doesn't really feel optimal. Mailer has a similar list [here](https://github.com/mozilla/fxa-auth-server/blob/master/lib/senders/templates/index.js#L59) that has to be updated when a new template is added.